index.html

```diff
@@ -5931,7 +5931,12 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content is supplementary to some other piece of content.</p>
+				<p>In cases where an element with role <rref>note</rref> has an explicit association with another element, authors can use one of the following mechanisms to point to the note from the associated element:</p>
+				<ul>
+					<li>If the note is structured (for example with rich text, or operable features such as a close button), use <pref>aria-details</pref>.</li>
```

Comment:
this should probably say "If the note contains..."  rather than "is structured".  

Are people commonly using notes with close buttons?  Is this going to confuse people to think that a "note" is a "notification" - a notification generally being more appropriate to inject into an `alert` or `status` element, arguably a `region` landmark to promote its importance?

Reply:
Note and notification are not adequately distinguished from one another right now. And that's a problem.

I work on range of RIAs which all include notes with close buttons. **They are not websites, they are RIAs**

Whether this is **common** seems irrelevant to the fact that it is a genuine, real world case that's not forbidden by the spec. Such a thing is not unusual when a new feature is added to an existing UI - there will be a kind of 'sticky' tooltip with a close button drawn next to the new UI element with a label along the lines of "You can try the new XYZ feature here". That's basically what we have.

Reply:
Thanks for the context.  So then again are these "notes" or "notifications"?   To be clear, I'm not saying that a close button inside of a note is wrong or bad.  I just want to make sure we don't confuse people to think notes = notifications, where it is more common for a notification to have a close button.

Reply:
I wouldn't say they're notifications.

I conceive of a notification as something which typically appears during use, triggered by some background process, rather than a mechanism for orienting a user about a feature of a user interface.

Our notes are drawn when the user encounters the UI for the first time - as a result of _their_ choice to begin the activity. They're "affordances", really. 

This kind of thing is quite common in games and simulations, especially to provide a little extra guidance about the UI in a "first run" situation, "training level" or "onboarding" phase.

Reply:
so "teaching ui" then.  thank you.  this is kinda helpful for another discussion i've been having.

Reply:
resolving this thread as I have made updates to the content to address my concerns here.